guides/plugins/plugins/framework/data-handling/entities-via-attributes.md

```diff
@@ -0,0 +1,411 @@
+---
+nav:
+  title: Entities via attributes
+  position: 1
+
+---
+
+# Entities via attributes
+
+Since shopware v6.6.3.0 it is possible to register entities via php attributes. This is a new way to register entities in Shopware 6. This guide will show you how to do this.
+
+## Define the entity
+
+First, you need to define your entity. This is done by creating a new class and adding the `Entity` attribute to it. The attribute requires the `name` parameter, which is the name of the entity. The `name` parameter is required and must be unique.
+
+Also, you have to extend the `EntityStruct` class and define a primary key. The primary key is defined by adding the `PrimaryKey` attribute to a property. In theory, the primary key can be of any type, but it is recommended to use a `UUID`. 
```

Comment:
`EntityStruct` is an alias of `Shopware\Core\Framework\DataAbstractionLayer\Entity`. IMO, it would make sense to de-alias it and use alias for `Shopware\Core\Framework\DataAbstractionLayer\Attribute\Entity as EntityAttribute` instead?